assets/aml-benchmark/components/src/aml_benchmark/utils/io.py

```diff
@@ -206,3 +208,24 @@ def save_list_to_jsonl_if_path_provided(data: List[Any], path: Optional[str]):
     if path:
         df = pd.DataFrame(data)
         df.to_json(path, orient='records', lines=True)
+
+
+def read_files(file_paths: List[str]) -> List[Dict[str, Any]]:
```

Comment:
nit: Can we rename it to read_jsonl_files since it will only work for jsonl files else throw an error?

Reply:
Here is a better way to do things without creating confusion.

This function should be renamed to `read_jsonl_file` which will take a single `file_path` and return `dict`. Call this from within `read_jsonl_files`.

Reply:
Removed this function.